WebAPI/Controllers/RolesAndPermissionsController.cs

```diff
@@ -17,6 +20,56 @@ public RolesAndPermissionsController(IRolesAndPermissionsManager rolesAndPermiss
             _rolesAndPermissionsManager = rolesAndPermissionsManager;
         }
 
+        /// <summary>
+        ///     Get All Available Roles
+        /// </summary>
+        /// <response code="200"> Available Roles</response>
+        /// <response code="403">UnAuthorized Access </response>
+        [ProducesResponseType(200, Type = typeof(IEnumerable<Role>))]
+        [ProducesResponseType(403, Type = null)]
```

Comment:
When exactly do the code return 403 ?????
This is a recurring error in your code, please review the entire controller